### PR TITLE
feat: add managed agents section for agentverse docs

### PR DESCRIPTION
**Pull request by FelixNicolaeBucsa**

### CODE REVIEW

File: pages/concepts/agentverse/managed-agents.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Hosting / Managed Agents
+
+In the **Managed Agents** section you are able to build any type of agent you wish. The benefit for you is that you do not have to carry the hassle of deploying your agents continuously, given that we run them for you using cloud technology. This means that your agents will continue running and receive messages without the need of your presence for them to be deployed. **The only task for you is to create great agents to satisfy all kinds of your needs!**
```
Comment (devjsc) [inline pages/concepts/agentverse/managed-agents.md]: merge this into hosting.mdx - hosting.mdx is the example of how we'd like concepts to be written

**Review by devjsc — CHANGES_REQUESTED** [review]
I think this is unnecessary as covered in hosting.mdx

**Comment by FelixNicolaeBucsa** [comment]
Ok, no problem

File: pages/concepts/agentverse/managed-agents.md (hunk 1 above)
Comment (FelixNicolaeBucsa) [reply pages/concepts/agentverse/managed-agents.md]: Ok, perfect. Will have a view and edit all Agentverse related docs in the same way

**Review by devjsc — APPROVED** [review]

**Pull request merged**